train-core.gemspec

```diff
@@ -30,10 +30,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'json', '>= 1.8', '< 3.0'
+  spec.add_dependency 'mixlib-shellout', '>= 2.0', '< 4.0'
   spec.add_dependency 'net-ssh', '>= 2.9', '< 6.0'
```

Comment:
This is left at 3.x since the ed25519 gem does not directly depend on net-ssh. If a person uses the old net-ssh they won't get ed25519 support, but nothing will break.